### PR TITLE
[Speed]Update how-do-i-enable-http2-server-push-in-wordpress.mdx

### PR DESCRIPTION
**Pull request by luishcardoso**
### Summary

For better visibility of the PHP code. Per docs style guide.

Thanks!

### CODE REVIEW

**Review by zeinjaber — COMMENTED** [review]
@nenizera - thanks for raising this.

The right syntax should be:

```php
define('CLOUDFLARE_HTTP2_SERVER_PUSH_ACTIVE', true);
```

**Comment by zeinjaber** [comment]
@nenizera - thanks for raising this.
The right syntax should be:
[
Screenshot 2024-11-14 at 11 09 12 AM
](url)

File: src/content/docs/support/third-party-software/content-management-system-cms/how-do-i-enable-http2-server-push-in-wordpress.mdx [hunk 1]
```diff
@@ -13,10 +13,11 @@ Article IDs: 115002816808 | How do I enable HTTP/2 Server Push in WordPress
 
 Cloudflare supports HTTP/2 Server Push and it can be enabled for stylesheets and scripts using Cloudflare’s WordPress plugin. In order to utilise this feature, you must first ensure you have the Cloudflare WordPress plugin [installed and set-up on your site](https://support.cloudflare.com/hc/en-us/articles/227634427-Using-Cloudflare-with-WordPress).
 
-Once the plugin is installed, you can enable HTTP/2 Server Push simply by adding the following configuration code to your *wp-config.php* file:
-
-*define('CLOUDFLARE\_HTTP2\_SERVER\_PUSH\_ACTIVE', true);*
+Once the plugin is installed, you can enable HTTP/2 Server Push simply by adding the following configuration code to your `wp-config.php` file:
 
+```php
+define('CLOUDFLARE\_HTTP2\_SERVER\_PUSH\_ACTIVE', true);
```
Comment (pedrosousa) [inline src/content/docs/support/third-party-software/content-management-system-cms/how-do-i-enable-http2-server-push-in-wordpress.mdx]:
```suggestion
define('CLOUDFLARE_HTTP2_SERVER_PUSH_ACTIVE', true);
```
Per Zein's suggestion.

File: src/content/docs/support/third-party-software/content-management-system-cms/how-do-i-enable-http2-server-push-in-wordpress.mdx [hunk 2]
```diff
@@ -13,10 +13,11 @@ Article IDs: 115002816808 | How do I enable HTTP/2 Server Push in WordPress
 
 Cloudflare supports HTTP/2 Server Push and it can be enabled for stylesheets and scripts using Cloudflare’s WordPress plugin. In order to utilise this feature, you must first ensure you have the Cloudflare WordPress plugin [installed and set-up on your site](https://support.cloudflare.com/hc/en-us/articles/227634427-Using-Cloudflare-with-WordPress).
 
-Once the plugin is installed, you can enable HTTP/2 Server Push simply by adding the following configuration code to your *wp-config.php* file:
-
-*define('CLOUDFLARE\_HTTP2\_SERVER\_PUSH\_ACTIVE', true);*
+Once the plugin is installed, you can enable HTTP/2 Server Push simply by adding the following configuration code to your `wp-config.php` file:
```
Comment (pedrosousa) [inline src/content/docs/support/third-party-software/content-management-system-cms/how-do-i-enable-http2-server-push-in-wordpress.mdx]:
```suggestion
Once the plugin is installed, you can enable HTTP/2 Server Push by adding the following line to your `wp-config.php` file:
```

File: src/content/docs/support/third-party-software/content-management-system-cms/how-do-i-enable-http2-server-push-in-wordpress.mdx [hunk 3]
```diff
@@ -13,10 +13,11 @@ Article IDs: 115002816808 | How do I enable HTTP/2 Server Push in WordPress
...
 
-Once the plugin is installed, you can enable HTTP/2 Server Push simply by adding the following configuration code to your *wp-config.php* file:
-
-*define('CLOUDFLARE\_HTTP2\_SERVER\_PUSH\_ACTIVE', true);*
+Once the plugin is installed, you can enable HTTP/2 Server Push by adding the following line to your `wp-config.php` file:
 
+```php
+define('CLOUDFLARE_HTTP2_SERVER_PUSH_ACTIVE', true);
+```
 You should insert this line above where it says *"/\* That's all, stop editing! Happy blogging. \*/*", like follows:|
```
Comment (pedrosousa) [inline src/content/docs/support/third-party-software/content-management-system-cms/how-do-i-enable-http2-server-push-in-wordpress.mdx:22]:
```suggestion

You should insert this line above where it says *"/\* That's all, stop editing! Happy blogging. \*/*", like follows:|
```

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**